robberies/fleeca-bank-robbery.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Fleeca Bank Robbery
+---
+
+**Fleeca Bank Robbery**  
+
+Adaugat un nou tip de jaf - jaf armat la banca. Jucatorii pot jefui micile banci din orasul Los Santos.  
+Acest tip de rob este unul mai complex, drept urmare jucatorii au nevoie de o arma cu munitie, C4 Bomb, Drill & Pliers.  
+
+Noile iteme pot fi dropate la job-urile de mai jos:  
+
+C4 Bomb se craftează pe insula Cayo Perico din:  
+ - 2x Explosive  
+ - 3x Wires  
+ - 1x Electronic Keyboard  
+
+- Trucker:
+    - Explosive: sansa de drop de 20% la fiecare $300 castigati in urma efectuarii unei curse ilegale    
+
+- Electrician: 
+    - Wires: sansa de drop de 20% la fiecare $300 castigati la job
+    - Electronic Keyboard: sansa de drop de 20% la fiecare $900 castigati la job
+
+
+- Tow Truck Driver:
+    - Drill: sansa de drop de 20% la fiecare $400 castigati la job
+
+- Uber:
+    - Pliers: sansa de drop de 20% la fiecare $400 castigati la job
+
+**Defasurarea unui jaf la banca**:  
+- Jucatorul taie curentul bancii de la panoul aflat in exteriorul acesteia  
+- Jucatorul ucide paznicul seifului  
+- Jucatorul deschide seiful utilizand C4 Bomb  
+- In momentul in care seiful este deschis jucatorii pot jefui cutiile de valori ce contin aur/argint aflate in acesta sau pot fura banii aflati pe masa.  
+
+In momentul in care un jucator jefuieste o cutie de valori sau fura banii acesta va primi automat wanted.  
+
+Rob-ul este considerat complet cand jefuiesti toate bunurile.
+
+Jucatorii primesc 90 Heat.  
+
+Reward-uri rob:  
+    - 2-7 Goldbars (se pot vinde la Pawn Shop sau transforma in bijuterii pe insula Cayo Perico)  
+    - 2-7 Silverbar (se pot vinde la Pawn Shop sau transforma in bijuterii pe insula Cayo Perico)  
+    - $8.000-$10.000 bani marcati (se pot schimba in bani normali pe insula Cayo Perico dacă clanul tău a deblocat perk-ul Money Laundering)  
```

Comment:
Ce primesti din jaf la banca / 'Fleeca Bank Robbery'.

/n Dupa un jaf la banca o sa primesti  $8.000-$10.000 bani marcati (se pot schimba in bani normali pe insula Cayo Perico dacă clanul tău a deblocat perk-ul Money Laundering),  2-7 Silverbar / Goldbar (se pot vinde la Pawn Shop sau transforma in bijuterii pe insula Cayo Perico) si 90 Heat Level.